hknweb/settings/prod.py

```diff
@@ -15,3 +14,13 @@
     'dev-hkn-eecs-berkeley-edu.apphost.ocf.berkeley.edu',
     'dev-hkn.eecs.berkeley.edu',
 ]
+
+# Static files (CSS, JavaScript, Images)
+# https://docs.djangoproject.com/en/2.1/howto/static-files/
+STATIC_URL = 'https://ocf.berkeley.edu/~hkn/'
+STATIC_ROOT = '/home/h/hk/hkn/public_html/hknweb/static/'
```

Comment:
I think this path should be ok for now, but if we make per-directory releases like what Capistrano did, then this might need to be moved somewhere else.

Reply:
My understanding is that STATIC_ROOT is the final resting place of static files after deployment, i.e. ready to be served. If we have a single unified folder for all release static files, then new releases will clobber old files (desirable?); idk what you meant but splitting this into `static/<release_timestamp>` would require symlinking to avoid path changes? (might also be desirable). A single folder would be cleaner but timestamped folders would be less breakage, at the cost of requiring more fabric maintenance.